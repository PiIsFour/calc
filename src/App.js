import React, { Component } from 'react'
import Calculator from './calculator/'

class App extends Component {
	render () {
		return (
			<div>
				<header>
					<h1>Welcome to Calc</h1>
				</header>
				<Calculator />
			</div>
		)
	}
}

export default App
