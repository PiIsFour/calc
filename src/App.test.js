import React from 'react'
import ReactDOM from 'react-dom'
import App from './App'
import { shallow } from 'enzyme'
import enzymeToJson from 'enzyme-to-json'

describe('app', () => {
	it('renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(<App />, div)
		ReactDOM.unmountComponentAtNode(div)
	})

	it('snapshot matches', () => {
		const wrapper = shallow(<App />)
		expect(enzymeToJson(wrapper)).toMatchSnapshot()
	})

	it('shows the calculator', () => {
		const wrapper = shallow(<App />)
		expect(wrapper.find('Calculator')).toExist()
	})
})
