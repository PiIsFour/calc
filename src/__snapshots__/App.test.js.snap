// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app snapshot matches 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <header>
    <h1>
      Welcome to Calc
    </h1>
  </header>
  <Calculator />
</div>
`;
